generate.pl

```diff
@@ -295,11 +295,14 @@ =head1 DESCRIPTION
     && {{test}} \
     && make install \
     && cd /usr/src \
-    && curl -LO {{cpanm_dist_url}} \
-    && echo '{{cpanm_dist_sha256}} *{{cpanm_dist_name}}.tar.gz' | sha256sum -c - \
+    && curl -fLO {{cpanm_dist_url}} \
+    && echo '{{cpanm_dist_sha256}} *{{cpanm_dist_name}}.tar.gz' | sha256sum --strict --check - \
     && tar -xzf {{cpanm_dist_name}}.tar.gz && cd {{cpanm_dist_name}} && perl bin/cpanm . && cd /root \
     && cpanm IO::Socket::SSL \
-    && cd /usr/local/bin && curl -LO https://raw.githubusercontent.com/skaji/cpm/0.997011/cpm && chmod +x cpm \
+    && cd /usr/local/bin \
+    && curl -fLO https://raw.githubusercontent.com/skaji/cpm/0.997011/cpm \
+    && echo '7dee2176a450a8be3a6b9b91dac603a0c3a7e807042626d3fe6c93d843f75610 *cpm' | sha256sum --strict --check - \
```

Comment:
```suggestion
    && curl -fLO https://raw.githubusercontent.com/skaji/cpm/0.997011/cpm \
    # sha256 checksum is from docker-perl team, cf https://github.com/docker-library/official-images/pull/12612#issuecomment-1158288299
    && echo '7dee2176a450a8be3a6b9b91dac603a0c3a7e807042626d3fe6c93d843f75610 *cpm' | sha256sum --strict --check - \
```

Reply:
Makes sense!  Updated in https://github.com/Perl/docker-perl/compare/e91398ad5050b401f936731dd3a7928a85cd0dca..a031faf5fa951b74d50960c194a0e0091b8bd490 :+1: